mlrun/api/api/endpoints/functions.py

```diff
@@ -706,13 +706,14 @@ def _build_function(
             reason=f"runtime error: {err_to_str(err)}",
         )
     try:
+        is_nuclio_runtime = bool(fn.kind in RuntimeKinds.nuclio_runtimes())
```

Comment:
```suggestion
        is_nuclio_runtime = fn.kind in RuntimeKinds.nuclio_runtimes()
```
I think the `bool(...)` is redundant